apps/web/src/pages/_app.tsx

```diff
@@ -6,15 +6,14 @@ import type { AppProps } from 'next/app';
 import { Footer, Header } from '@components';
 
 axios.defaults.baseURL = process.env.NEXT_PUBLIC_API_URL;
+console.log('backend url = ', axios.defaults.baseURL);
```

Comment:
Stray console.log, might be good to remove it.

Reply:
Removed.
I wonder why NEXT_PUBLIC_API_URL in .env.local is not applied after restarting by file changes. I need to restart the container.

Reply:
.env files typically are only applied on application start aren't they?